apps/demo/src/app/checkout/components/forms/address-form/components/address-form/address-form.component.html

```diff
@@ -1,39 +1,51 @@
 <div class="address-form">
```

Comment:
The style of the form looks off in checkout.

Reply:
fixed